doc/users/plotting/examples/colormap_normalizations_arbitrarynorm.py

```diff
@@ -0,0 +1,62 @@
+"""
```

Comment:
We should probably move all of those in the examples folder - else they won't be dealt with properly in the gallery.
The title + description element is a fairly recent thing - we'd like to use sphinx-gallery that requires such formatting..